translations/expansion_content.tex/pl.po

```diff
@@ -99,6 +99,13 @@ msgid ""
 "Spells with one School symbol on them are \\textbf{Basic Spells}, while Spells with four identical symbols are \\textbf{Expert Spells}. Certain game effects may affect only one of these types.\n"
 "\n"
 msgstr ""
+"\\subsection*{Szkoła Magii}\n"
+"Wiele rozszerzeń ma efekty, które odnoszą się do Szkół Magii\\index{Schools of Magic}.\n"
```

Comment:
jak myślisz, czy tu warto trzymać się oryginału en w postaci "Większość?"

```suggestion
"Wiele rozszerzeń ma efekty, które odnoszą się do Szkół Magii\\index{Schools of Magic}.\n"
```

Reply:
Myślę, że w sensie logicznym Wiele w odniesieniu do kart ma bardziej rzeczywiste odniesienie do stanu faktycznego; Wiekszosc owszem tez jest logicznie poprawne, ale znając przypadek z mojego poprzedniego tlumaczenia AI Heroes - to myślę, że ilość literek może mieć kolosalne znaczenie... a z podglądu już w tym momencie widać, że strona się rozjeżdża... i chyba trzeba szukać oszczędności w znakach ;)